### PR TITLE
Governance Poll 2024-08-12: LITE-PSM-USDC-A Phase 2 (Major Migration) Parameter Proposal

### PR DESCRIPTION
**Pull request by blimpa**
**LITE-PSM-USDC-A Phase 2 (Major Migration) Parameter Proposal**

Originating forum post: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839

@MakerJanSky, @0xldr, @VoteWizard

### CODE REVIEW

**Comment by VoteWizard** [comment]
Is there a reason for not including the "Keeper Network Threshold Parameters" section?

File: governance/polls/LITE-PSM-USDC-A Phase 2 Parameter Proposal - August 12, 2024.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - [DC-IAM `gap` (Target Available Debt)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-): Decrease by 180 million DAI, from 380 million DAI to **200 million DAI**.
+    - [DC-IAM `ttl` (Ceiling Increase Cooldown)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-): **12 hours** (remains unchanged).
+  - `LITE-PSM-USDC-A`
+    - [`tin`](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) & [`tout`](https://manual.makerdao.com/module-index/module-psm#fee-out-tout): **0** (remains unchanged).
+    - [DC-IAM `line` (Maximum Debt Ceiling)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-): Increase by 7.45 billion DAI, from 50 million DAI to **7.5 billion DAI**.
+    - [DC-IAM `gap` (Target Available Debt)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-): Increase by 180 million DAI, from 20 million DAI to **200 million DAI**.
+    - [DC-IAM `ttl` (Ceiling Increase Cooldown)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-): Set to **12 hours** (remains unchanged).
```
Comment (0xldr) [inline governance/polls/LITE-PSM-USDC-A Phase 2 Parameter Proposal - August 12, 2024.md:55]: Matches PSM-USDC-A entry

```suggestion
    - [DC-IAM `ttl` (Ceiling Increase Cooldown)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-): **12 hours** (remains unchanged).
```

File: governance/polls/LITE-PSM-USDC-A Phase 2 Parameter Proposal - August 12, 2024.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - [`buf`](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8): Increase by 180 million DAI, from 20 million DAI to **200 million DAI**.
+- [GSM Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay): Increase by 14 hours, from 16 hours to **30 hours**.
+  - Update [Governance Scope, article 10.1.A](https://mips.makerdao.com/mips/details/MIP113#10-1-1a) accordingly.
+
+- A new KeeperJob will be deployed with the following parameters:
+    - `fill`: The `rushThreshold` will be set to **20M DAI**
+    - `trim`: The `gushThreshold` will be set to **20M DAI**
+    - `chug`: The `cutThreshold` will be set to **300K DAI**
```
Comment (0xldr) [inline governance/polls/LITE-PSM-USDC-A Phase 2 Parameter Proposal - August 12, 2024.md:63]: Improve clarity and consistency

```suggestion
    - `fill`: The `rushThreshold` will be set to **20 million DAI**
    - `trim`: The `gushThreshold` will be set to **20 million DAI**
    - `chug`: The `cutThreshold` will be set to **300,000 DAI**
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Review by 0xldr — APPROVED** [review]

**Review by MakerJanSky — APPROVED** [review]
Nice work.

**Pull request merged**